examples/dynamo_integration/README.md

```
@@ -140,6 +148,25 @@ python scripts/plot_throughput_vs_tsq_per_request.py \
+nat eval --config_file examples/dynamo_integration/react_benchmark_agent/configs/eval_config_rethinking_full_test.yml
+
+# Compare performance across all jobs
+python scripts/plot_throughput_vs_tsq_per_request.py \
+  ./react_benchmark_agent/outputs/dynamo_evals/banking_data_eval_full_test/jobs/
+```
+
+This script will generate comparative visualizations showing throughput vs. Tool Selection Quality (TSQ) metrics across all jobs in the specified directory, allowing you to analyze the performance differences between different agent configurations.
+
+> [!NOTE]
+> **Multi-Backend Comparisons**: Evaluation runs can be performed across multiple Dynamo backend configurations (e.g., different routing strategies, tensor parallelism settings, or hardware configurations) and compared using the same script. Simply run evaluations against different Dynamo deployments and place the results in the same jobs directory for side-by-side analysis.
+
```

markdownlint-cli2 (0.20.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)